src/Raven.Server/Rachis/RachisLogHistory.cs

```diff
@@ -134,9 +134,6 @@ public void InsertHistoryLog(ClusterOperationContext context, long index, long t
             if (guid == null) // shouldn't happened in new cluster version!
                 return;
 
-            if (guid == RaftIdGenerator.DontCareId)
```

Comment:
You can't do that.

A command is uniquely defined by this guid, so All `DontCareId` commands will have the same string (the empty one) and will be considered as the same command... which will break everything :)

Reply:
Although I see all tests has passed.. this is because you only removed it here in the `InsertHistoryLog` so this probably will not be updated to the status of `Committed`.. 

Maybe we can generate a stable id here... let's say something like `DontCare/{index}`?

Reply:
Done